python/paddle/fluid/transpiler/common.py

```diff
@@ -0,0 +1,74 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
The name is `common.py` while it's not quite "common", see https://github.com/PaddlePaddle/Paddle/issues/9770#issuecomment-379840606 for details

Reply:
Thanks! `common.py` is only used by `distribute_transpiler`, so I delete it and reuse it for future.